trulens_eval/examples/trubot/trubot_example.ipynb

```diff
@@ -45,34 +70,47 @@
    "outputs": [],
    "source": [
     "from pprint import PrettyPrinter\n",
-    "from typing import Sequence\n",
-    "import numpy as np\n",
     "\n",
     "# Imports from langchain to build app:\n",
     "from langchain.chains import ConversationalRetrievalChain\n",
     "from langchain.embeddings.openai import OpenAIEmbeddings\n",
     "from langchain.llms import OpenAI\n",
     "from langchain.memory import ConversationSummaryBufferMemory\n",
-    "from langchain.vectorstores import DocArrayHnswSearch\n",
+    "import numpy as np\n",
+    "\n",
+    "# Imports various API keys from .env:\n",
+    "from trulens_eval.keys import *\n",
+    "\n",
+    "# Check to make sure the required api keys have been set.\n",
+    "assert \"OPENAI_API_KEY\" in locals(), \"This example uses the openai API and needs an API key. See .env example on how to set it.\"\n",
```

Comment:
Can we have these be set via the environment variables instead of requiring a .env file? It's good practices, but not standard for users to have a .env set up.
See this tutorial from pinecone on how they do it: https://colab.research.google.com/github/pinecone-io/examples/blob/master/generation/langchain/handbook/05-langchain-retrieval-augmentation-fast.ipynb

Essentially they ask you to either set it in the notebook itself or set it as an os environment variable.

`# find API key in console at app.pinecone.io
PINECONE_API_KEY = os.getenv('PINECONE_API_KEY') or 'PINECONE_API_KEY'
# find ENV (cloud region) next to API key in console
PINECONE_ENVIRONMENT = os.getenv('PINECONE_ENVIRONMENT') or 'PINECONE_ENVIRONMENT'

OPENAI_API_KEY = os.getenv('OPENAI_API_KEY') or 'OPENAI_API_KEY'
`

Reply:
Did this:
```
from trulens_eval.keys import setup_keys

setup_keys(
    OPENAI_API_KEY="fill this in if not in your environment",
    HUGGINGFACE_API_KEY='fill this in if not in your environment'
)
```

The `setup_keys` there automatically gets keys from either .env or the environment if the user does not provide it in the string and will raise an error if none of the sources provided the keys.

Reply:
This makes sense.